### DISCUSSION TITLE
Logout of web interface

### DISCUSSION THREAD

**Discussion by Forts117** [chuot/rdio-scanner#377] [General]
Stupid question... how do I logout of the web interface so I can enter a different access code?

**Comment by Millecactus** [comment]
Hi,
What do you means by "web interface" ?
If you means that you want to logout an user created in your JAVA code, you can add a function "logout()" in your User Class. This method can be as simple as resetting the user's attribute values ​​or making them null.
Then, call the "logout()" method when you want to log out of the current user.
